_sass/base/_markdown.scss

```diff
@@ -0,0 +1,92 @@
+.markdown {
+  padding-bottom: 60px;
+  
+  // Text Blocks
+  p {
+    margin: 10px 0;
+    font-weight: 300;
+
+    @media (min-width: $tablet) {
+      margin: 20px 0;
+    }
+  }
+  
+
+  img {
+    width: 100%;
+  }
+
+  // List Items
+  li {
+    line-height: 24px;
+  }
+
+  // Lists
+  ol,
+  ul {
+    list-style: initial;
+    list-style-position: inside;
+    color: $black;
+    font-size: 15px;
+    margin: 0 20px;
+
+    @media (max-width: 768px) {
+      margin: 0;
+    }
+  }
+
+  h1 {
+    color: $secondBrandColor;
+  }
+
+  // Section Headlines 
+  h2 {
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Headlines 
+  h3 {
+    color: $black;
+    font-size: 18px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Subheadlines
+  h4 {
+    margin: 20px 25px;
+    font-size: 16px;
+    font-weight: 300;
+    color: $dark-gray;
+    
+    @media (max-width: 768px) {
+      margin: 20px 0;
+    } 
+    // Text immediately following h6 should have same level of indention  
```

Comment:
Thanks for the clarification @LisaFC . How does this look? H4 and their follow up paragraphs are now inline with other text elements. With the change, I believe there would be two levels of indention - One for most html elements, and one level of indention deeper for lists and code blocks

![screen shot 2016-07-18 at 3 40 05 pm](https://cloud.githubusercontent.com/assets/11986887/16932868/41f4dffc-4cfe-11e6-844d-16dbf779f1f8.png)